node/src/Pos/WorkMode.hs

```diff
@@ -19,57 +19,59 @@ module Pos.WorkMode
-import           Pos.Slotting.Impl.Sum  (currentTimeSlottingSum,
-                                         getCurrentSlotBlockingSum,
-                                         getCurrentSlotInaccurateSum, getCurrentSlotSum)
-import           Pos.Slotting.MemState  (HasSlottingVar (..), MonadSlotsData)
-import           Pos.Ssc.Class.Helpers  (SscHelpersClass)
-import           Pos.Ssc.Class.Types    (SscBlock)
-import           Pos.Ssc.Extra          (SscMemTag, SscState)
-import           Pos.Txp.MemState       (GenericTxpLocalData, TxpHolderTag)
-import           Pos.Util               (Some (..))
-import           Pos.Util.JsonLog       (HasJsonLogConfig (..), JsonLogConfig,
-                                         jsonLogDefault)
-import           Pos.Util.LoggerName    (HasLoggerName' (..), getLoggerNameDefault,
-                                         modifyLoggerNameDefault)
-import           Pos.Util.OutboundQueue (EnqueuedConversation (..), OQ)
-import qualified Pos.Util.OutboundQueue as OQ.Reader
-import           Pos.Util.TimeWarp      (CanJsonLog (..))
-import           Pos.Util.UserSecret    (HasUserSecret (..))
-import           Pos.Util.Util          (postfixLFields)
-import           Pos.WorkMode.Class     (MinWorkMode, TxpExtra_TMP, WorkMode)
+import           Pos.KnownPeers          (MonadFormatPeers (..), MonadKnownPeers (..))
+import           Pos.Network.Types       (HasNodeType (..), getNodeTypeDefault)
+import           Pos.Reporting           (HasReportingContext (..))
+import           Pos.Shutdown            (HasShutdownContext (..))
+import           Pos.Slotting.Class      (MonadSlots (..))
+import           Pos.Slotting.Impl.Sum   (currentTimeSlottingSum,
+                                          getCurrentSlotBlockingSum,
+                                          getCurrentSlotInaccurateSum, getCurrentSlotSum)
+import           Pos.Slotting.MemState   (HasSlottingVar (..), MonadSlotsData)
+import           Pos.Ssc.Class.Helpers   (SscHelpersClass)
+import           Pos.Ssc.Class.Types     (SscBlock)
+import           Pos.Ssc.Extra           (SscMemTag, SscState)
+import           Pos.Txp.MemState        (GenericTxpLocalData, TxpHolderTag)
+import           Pos.Util                (Some (..))
+import           Pos.Util.JsonLog        (HasJsonLogConfig (..), JsonLogConfig,
+                                          jsonLogDefault)
+import           Pos.Util.LoggerName     (HasLoggerName' (..), getLoggerNameDefault,
+                                          modifyLoggerNameDefault)
+import           Pos.Util.OutboundQueue  (EnqueuedConversation (..), OQ)
+import qualified Pos.Util.OutboundQueue  as OQ.Reader
+import           Pos.Util.TimeWarp       (CanJsonLog (..))
+import           Pos.Util.UserSecret     (HasUserSecret (..))
+import           Pos.Util.Util           (postfixLFields)
+import           Pos.WorkMode.Class      (MinWorkMode, TxpExtra_TMP, WorkMode)
```

Comment:
I'd really prefer if we did not do these mass whitespace changes. It makes it very easy to create conflicts and very hard to spot changes (which makes resolving conflicts time consuming and error prone).